src/ProgressOnderwijsUtils/SchemaReflection/CheckConstraint.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Linq;
+using Microsoft.Data.SqlClient;
+using static ProgressOnderwijsUtils.SafeSql;
+
+namespace ProgressOnderwijsUtils.SchemaReflection
+{
+    public sealed partial class DatabaseDescription
+    {
+        public struct CheckConstraintEntry : IWrittenImplicitly
+        {
+            public DbObjectId CheckConstraintObjectId { get; set; }
+            public string Name { get; set; }
+            public string Definition { get; set; }
+            public DbObjectId TableObjectId { get; set; }
+        }
+
+        public sealed class CheckConstraint
+        {
+            readonly DatabaseDescription db;

```

Comment:
Vraag me af of dit wenselijk is. De description bevat de check-contarints, en de check-constraint de description. Is net even een te clevere cyclische referentie.